CONTRIBUTING.md

```diff
@@ -114,7 +114,6 @@ Of course, there is also the possibility to add existing SVGs. Mostly, however,
 
 ### Known icon sources
 
-- [Material Design Icons](https://materialdesignicons.com/)
```

Comment:
I'd like to keep it, to have more than one souce of inspiration how icon can look like. In the past "https://materialdesignicons.com/" was one of my main sources to take a look on how icons can be created for this extension. Why not just keep the link here? If you want we can update the link to the new URL: https://pictogrammers.com/library/mdi/

Reply:
What's the point?
https://fonts.google.com/icons has all types of icons, sliders, flexible settings, everything you need

Reply:
materialdesignicons.com is a superset of the Google icons. It has even more icons which could be used. Having more than one link provides more options.

Reply:
why we can't recommend materialdesignicons.com? What does it disqualify to be not recommended?

Reply:
> why we can't recommend materialdesignicons.com? What does it disqualify to be not recommended?

Because the second one has more sliders, settings, and a better interface

Reply:
I would just add both links. Both serves the purpose of a recommended source: to look for icons that fit into the theme. The only problem I find with the "new" Material Symbols is that they are outlined:


**Material Symbols**

![image](https://github.com/user-attachments/assets/73a88ba2-453a-4588-9e7a-776e65a7fc88)

vs

**Legacy Material Design Icons**

![image](https://github.com/user-attachments/assets/ce573152-3472-48de-9d31-f29ab3fbf1f9)

We can use material symbols for shapes, but not exactly as they are, because most motives in this theme are filled.

But that said, both are great sources IMO

Reply:
> The only problem I find with the "new" Material Symbols is that they are outlined:

Lol

![image](https://github.com/user-attachments/assets/100a5930-b852-4359-b8b2-ac16c1e35d11)

Reply:
Here you can choose any icons, old, new, filled, outlined, not outlined, any, without the need to switch to another site

<img src="https://github.com/user-attachments/assets/d57bb195-299c-4160-afd9-6aa51b364337" width="400"></p>